packages/select-react/src/Select.tsx

```diff
@@ -70,19 +72,19 @@ export function Select({
     width,
     ...selectProps
 }: Props) {
-    const [selectedValue, setSelectedValue] = useState(value);
-    const [internalFocus, setInternalFocus] = useState(false);
-    const hasSelectedValue = typeof selectedValue !== "undefined" && selectedValue !== "";
+    const [searchValue, setSearchValue] = useState("");
+    const hasSelectedValue = typeof value !== "undefined" && value !== "";
 
-    const getLabelFromValue = useCallback(
-        (value: string | undefined) => {
-            const matchingItem = items.map(getValuePair).filter((item) => item.value === value)[0];
-            return matchingItem && matchingItem.label;
-        },
-        [items],
-    );
-    const [displayedValue, setDisplayedValue] = useState(getLabelFromValue(value));
+    const itemsWithVisibility = items.map(getValuePair).map((item) => {
```

Comment:
```suggestion
    const visibleItems = items.map(getValuePair).map((item) => {
```

Litt pirk, men "with visibility" betyr at de kan se noe mens "visible" betyr at de er synlige.

Reply:
Endret det 👍